.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -27,13 +27,25 @@ The MLflow Community encourages bug fix contributions. Would you or another memb
 ### Describe the problem
 Describe the problem clearly here. Include descriptions of the expected behavior and the actual behavior.
 
+For issues related to the tracking features (e.g. mlflow doesn't log a run in my database), please insert the following code in your python script / notebook where you encountered the bug and run it:
+
+```python
+print("MLflow version:", mlflow.__version__)
```

Comment:
It might be nice to have the mlflow server cli command with a note for users to strip out the host name (or we could do that for them with urllib.urlparse and setting 'sensitive' values to <hostname>/<root>. WDYT?

Reply:
Great suggestion! I think we could just ask users to manually maks sensitive information.